vpr/src/place/place.cpp

```diff
@@ -1569,7 +1568,7 @@ static e_move_result try_swap(const t_annealing_state* state,
         calculate_reward_and_process_outcome(placer_opts, move_outcome_stats,
                                              delta_c, timing_bb_factor, move_generator);
     } else {
-//        std::cout << "Group move delta cost: " << delta_c << std::endl;
+        //        std::cout << "Group move delta cost: " << delta_c << std::endl;
```

Comment:
remove the else statement
I used it for debugging, but it somehow found its way into master